templates/command/arrival.txt

```diff
@@ -0,0 +1,20 @@
+[center][solcrest][u][large]Welcome to the SEV Torch![/large][/u][eclogo][/center]
+[hr]
+Whether from shore, cryogenic storage, or by other means, you've now found yourself on the SEV Torch.
```

Comment:
Canonically, people are only expected to arrive on the Torch when it stops in port, a small number of crew board or depart, and it keeps going. That's my understanding, at least. Ingame, this would probably be given to:

- very new players playing CA or XPL who, in-universe, have just arrived on the Torch a few days or weeks ago

- people rescued from offsite spawns, offsite antagonists doing something weird, etc

So I would adjust this language to reflect the situation the Torch *expects* people to arrive in: volunteers from the EC or Fleet or EXO contractors who signed up for this position or were transferred from another position.

This line specifically I would just delete.

```suggestion

```